### PR TITLE
Meta: Add link to tc39/template-for-proposals

### PR DESCRIPTION
**Pull request by sffc**
I couldn't find a link to this in the contributing doc.  I think it is a useful link.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ To make a feature request, document the problem and a sketch of the solution wit
 
 Your goal will be to convince others that your proposal is a useful addition to the language and recruit TC39 members to help turn your request into a proposal and shepherd it into the language. Once a proposal is introduced to the committee, new features are considered by the committee according to the [TC39 process document](https://tc39.es/process-document/).
 
+Proposals should be backed by a repository created from the [TC39 Template for Proposals](https://github.com/tc39/template-for-proposals). Once adopted to Stage 1 by the committee, the proposal repository should be transferred into the tc39 org on GitHub.
```
Comment (ljharb) [inline CONTRIBUTING.md:40]: I'm not sure we have consensus that proposals "should" be created from the template; that's there more as a convenient option.

Comment (sffc) [reply CONTRIBUTING.md]: I changed "should" to "may"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,6 +37,8 @@ To make a feature request, document the problem and a sketch of the solution wit
 
 Your goal will be to convince others that your proposal is a useful addition to the language and recruit TC39 members to help turn your request into a proposal and shepherd it into the language. Once a proposal is introduced to the committee, new features are considered by the committee according to the [TC39 process document](https://tc39.es/process-document/).
 
+Proposals may be backed by a repository created from the [TC39 Template for Proposals](https://github.com/tc39/template-for-proposals). Once adopted to Stage 1 by the committee, the proposal repository should be transferred into the tc39 org on GitHub.
```
Comment (bakkot) [inline CONTRIBUTING.md:40]: How about this?

```suggestion
You can look at [existing proposals](https://github.com/tc39/proposals/) for examples of how proposals are structured, and some delegates use [this template](https://github.com/tc39/template-for-proposals) when creating repositories for their proposals. Proposals need to have a repository and be moved to the TC39 org on GitHub once they reach Stage 1.
```

Comment (sffc) [reply CONTRIBUTING.md]: Works for me

**Review by syg — APPROVED** [review]
We talked about this on the editor call. Let's land this for now but let's move most of this info to the how-we-work repo in the future and link to how-we-work.

**Review by ljharb — APPROVED** [review]

**Pull request merged**